Add Secret key renewal azure app info (#1104)
MicrosoftDocs/SCCMdocs: sccm/core/servers/deploy/configure/azure-services-wizard.md

Select **OK** to create the native app in Azure AD and close the Create Client Application dialog. This action returns to the [Client App dialog](#client-app-dialog).

### Renew secret key Azure AD apps

> Prior to 1806 version, to renew secret key of an Azure App, you need to recreate the App.

Reviewer: A couple of minor issues on this line:
- Add "the" before "secret key": "...to renew the secret key..."
- lowercase "app" (both instances)

```suggestion
Prior to 1806 version, to renew the secret key of an Azure app, you need to recreate the app.
```

> On 1806 and above:

Reviewer: Please change "above" to "later". We want to use terms that are strictly time-based, and don't imply levels or preferred.

```suggestion
On version 1806 and later:
```

> Created App --> renewal will be done by entering azure credentials, and adding the info on the renew secret key wizard

Reviewer: Let's do these as a bulleted list, and use a colon instead of the arrow. I don't have access to a console right now, but isn't there an option in the ribbon to renew key? We should explicitly mention that, in bold. For example:

```suggestion
- Created app: Go to the **Azure AD** node, and select the tenant. In the details pane, select the app, and then select **Renew Secret Key** in the ribbon. Entering your Azure credentials, and add the info on the renew secret key wizard.
```

Reviewer: Sorry I should have been more clear. A bulleted list starts with a dash: `-`
For example:
```
- Created app: ...  
- Imported app: ...  
```

Reviewer: Also, please capitalize "Azure" at the end of line 216:
"Enter your Azure credentials"

> Imported App --> Get new secret key and expiry date from Azure Portal

Reviewer: Like the line above, make it a bulleted list with a colon instead of an arrow. Also "portal" is not capitalized.

Select **OK** to create the native app in Azure AD and close the Create Client Application dialog. This action returns to the [Client App dialog](#client-app-dialog).

### Renew secret key Azure AD apps
Prior to 1806 version, to renew the secret key of an Azure app, you need to recreate the app.
On 1806 and later:
+ Created app:  Under **Cloud Services node** go to **Azure Active Directory Tenants**, on details pane, select the Tenant on which the application was created, and select **Renew Secret Key**. Enter azure credentials.
+ Imported app:  Renew and note the new secret key and expiry date from Azure portal. Add this informantion on the **Renew Secret Key** wizard.

>[!Note]

> > Save secret key before closing azure applicaiton properties **key** page. This info will be removed when page is closed.

Reviewer: On this line, capitalize "Azure" and there's a typo with spelling of "application"
